docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.info.yml

```diff
@@ -25,3 +28,8 @@ regions:
   sidebar_first: "First sidebar"
 regions_hidden:
   - sidebar_first
+
+components:
+  namespaces:
+    humsci_basic: ../humsci_basic/templates
+    patterns: ../../../modules/humsci/hs_layouts/patterns
```

Comment:
```suggestion
```

These should be defined in the `info.yml` files for the module and theme instead, and then the namespaces should already be available to extend here. `hs_layouts` already has an `hs_layouts` namespace defined that can be used to extend the templates: https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/modules/humsci/hs_layouts/hs_layouts.info.yml

https://www.drupal.org/docs/contributed-modules/components/registering-twig-namespaces#s-extending-a-twig-namespace
https://www.drupal.org/docs/develop/theming-drupal/twig-in-drupal/extending-templates

Reply:
Fixed!